code/js/modules/BaseController.js

```diff
@@ -214,7 +209,12 @@
       song: this.getSongData(this.selectors.song),
       artist: this.getSongData(this.selectors.artist),
       isPlaying: this.isPlaying(),
-      siteName: this.siteName
+      siteName: this.siteName,
+      canDislike: !!(this.selectors.dislike && this.doc().querySelector(this.selectors.dislike)),
+      canPlayPrev: !!(this.selectors.playPrev && this.doc().querySelector(this.selectors.playPrev)),
+      canPlayPause: !!((this.selectors.playPause && this.doc().querySelector(this.selectors.playPause)) || (this.selectors.play && this.doc().querySelector(this.selectors.play)) || (this.selectors.pause && this.doc().querySelector(this.selectors.pause))),
```

Comment:
Sure is :). After this is merged can you create a branch + PR for your `hasPlayer` changes? I can handle fixing that bug you were talking about. TY!